src/common/dao/artifact.go

```diff
@@ -26,6 +26,9 @@ import (
 func AddArtifact(af *models.Artifact) (int64, error) {
 	now := time.Now()
 	af.CreationTime = now
```

Comment:
The `CreationTime` is the time when the artifact is built, so it shouldn't be set here.

Reply:
It should be set here, the CreationTime is the time to insert data to db.

If you want the build time of the image, i think `BuildTime` need to be added like `PushTime` and `PullTime`

Reply:
The assignment for `PullTime` was removed, but `PushTime` was kept for the time to create artifact is the most time to push the image to registry.

Reply:
The build time of the artifact problem can create an issue.